cypress/integration/dib-travel/company-settings/travel-policy.spec.ts

```diff
@@ -0,0 +1,197 @@
+  });
+
+  it('submits empty travel policy form', () => {
+    cy.get('dib-company-management dib-travel-policy ui-button[type=primary]').click();
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog ui-button[type=success]').click();
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog').should('be.visible');
+  });
+
+  it('creates a new flight travel policy', () => {
+    cy.get('dib-company-management dib-travel-policy ui-button[type=primary]').click();
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog .dib-select').select(
+      travelPolicyForm.travelPolicyTypeFlight
+    );
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[name=name]').type(travelPolicyForm.travelPolicyName);
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[name=numberOfDaysInAdvance]').type(
+      travelPolicyForm.numberOfDaysInAdvance
+    );
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog dib-checkbox').click({ multiple: true });
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[name=budget]').type(
+      travelPolicyForm.travelPolicyBudget
+    );
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[name=budgetException]').type(
+      travelPolicyForm.travelPolicyBudgetException
+    );
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[name=flightDurationBudgetException]').type(
+      travelPolicyForm.flightDurationBudgetException
+    );
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog .add-btn').click();
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[data-placeholder=From]').type(
+      travelPolicyForm.flightFrom
+    );
+    cy.get('.cdk-overlay-container .cdk-overlay-pane .flight-search__result__text')
+      .contains(travelPolicyForm.flightFrom)
+      .click();
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[data-placeholder=To]').type(
+      travelPolicyForm.flightTo
+    );
+    cy.get('.cdk-overlay-container .cdk-overlay-pane .flight-search__result__text')
+      .contains(travelPolicyForm.flightTo)
+      .click();
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog dib-list-item select').select(travelPolicyForm.typeOfTrip);
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog dib-list-item input[name=budget]').type(
+      travelPolicyForm.travelPolicyBudget
+    );
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog input[placeholder=Search]').type(
+      travelPolicyForm.employeeName
+    );
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog .members').click();
+    cy.get('.cdk-overlay-container dib-travel-policy-dialog ui-button[type=success]').click();
+    cy.get('dib-company-management dib-travel-policy dib-expandable-item .section__header__title').should(
+      'contain',
+      travelPolicyForm.travelPolicyName
+    );
+  });
+
+  it('deletes a flight travel policy', () => {
```

Comment:
This test failed in 2nd pass. Please revisit.

```
Timed out retrying after 4050ms: cy.click() failed because this element is detached from the DOM.

<button _ngcontent-wsf-c10="" class="ui-button dib-outline dib-button-warning dib-button-size-sm" ng-reflect-klass="ui-button" ng-reflect-ng-class="[object Object]">delete</button>

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  > cy.contains()

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element

You typically need to re-query for the element or add 'guards' which delay Cypress from running new commands
```